Arduino/victoria_teensy_ros_node/victoria_teensy_ros_node.ino

```diff
@@ -513,4 +510,14 @@ void doMagnetometerDebug() {
   }
   ros_magnetometer_debug_pub.publish(&ros_debug_msg);
 }
-#endif
+
+/*
+ * Converts a frequency in hertz to the
+ * number of milliseconds to match the
+ * frequency. For example, 2 hz = 500 millis
+ * or 100 hz = 10 millis.
+ */
+int convertFreqToMillis(int frequency) {
+  return (int)((1.0/frequency) * 1000);
```

Comment:
If possible (I think it is), change to:
` return static_cast<int>((1.0/frequency) * 1000);`
If we can manage to stick to C++ style casting, we should